api/handlers/containerhandler.py

```diff
@@ -64,7 +65,7 @@ class ContainerHandler(base.RequestHandler):
             'parent_storage': containerstorage.ProjectStorage(),
             'storage_schema_file': 'session.json',
             'payload_schema_file': 'session.json',
-            'list_projection': {'info': 0, 'analyses': 0},
+            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0, 'subject.lastname': 0},
```

Comment:
Um, maybe? Dropping arbitrary-length info & analyses fields make sense to me here, adding subject fname / lastname seems more like a policy decision that doesn't jive at this layer.

But, you're a lot more native to this area than I am - wanna just document why those keys are dropped & go ahead with it being there?